src/sources/sqlite/sqlite-schema.lisp

```diff
@@ -3,66 +3,60 @@
 ;;;
 (in-package :pgloader.source.sqlite)
 
-(defvar *sqlite-db* nil
-  "The SQLite database connection handler.")
+(defclass copy-sqlite (db-copy)
+  ((db :accessor db :initarg :db))
+  (:documentation "pgloader SQLite Data Source"))
 
-;;;
-;;; Integration with the pgloader Source API
-;;;
-(defclass sqlite-connection (fd-connection)
-  ((has-sequences :initform nil :accessor has-sequences)))
-
-(defmethod initialize-instance :after ((slconn sqlite-connection) &key)
-  "Assign the type slot to sqlite."
-  (setf (slot-value slconn 'type) "sqlite"))
-
-(defmethod open-connection ((slconn sqlite-connection) &key check-has-sequences)
-  (setf (conn-handle slconn)
-        (sqlite:connect (fd-path slconn)))
-  (log-message :debug "CONNECTED TO ~a" (fd-path slconn))
-  (when check-has-sequences
-    (let ((sql (format nil (sql "/sqlite/sqlite-sequence.sql"))))
-      (log-message :sql "SQLite: ~a" sql)
-      (when (sqlite:execute-single (conn-handle slconn) sql)
-        (setf (has-sequences slconn) t))))
-  slconn)
+(defmethod initialize-instance :after ((source copy-sqlite) &key)
+  "Add a default value for transforms in case it's not been provided."
+  (let* ((transforms (when (slot-boundp source 'transforms)
+		       (slot-value source 'transforms))))
```

Comment:
This line mixes tabs and spaces. Also, ditto.

Reply:
It turns out that this code is a vestige from old times. Nowadays the cast function is taken care of at the `process-catalog` level when preparing our work specification, and then the catalog bits are copied over to the COPY objects. I removed the `initialize-instance :after` methods, we don't need them anymore.